solutions/standard/main.tf

```diff
@@ -119,6 +119,38 @@ module "secrets_manager" {
   cbr_rules                        = var.cbr_rules
 }
 
+module "secrets_group" {
+  count                    = var.existing_secrets_manager_crn == null ? 1 : 0
+  source                   = "terraform-ibm-modules/secrets-manager-secret-group/ibm"
+  version                  = "1.2.2"
+  region                   = local.secrets_manager_region
+  secrets_manager_guid     = local.secrets_manager_guid
+  secret_group_name        = var.default_secret_group_name
+  secret_group_description = "Default secrets group"
+  endpoint_type            = "private"
+}
+
+module "iam_service_access_group" {
+  count             = var.existing_secrets_manager_crn == null ? 1 : 0
```

Comment:
I think we need a boolean to allow users to opt out of the access group creation if they want